Mesh_3/include/CGAL/Mesh_3/polylines_to_protect.h

```diff
@@ -335,8 +363,8 @@ namespace CGAL {
   CGAL_assertion(nb_facets == expected_nb_facets);
   CGAL_USE(nb_facets); CGAL_USE(expected_nb_facets);
 
-
-  internal::Mesh_3::split_in_polylines(graph, polylines, K());
+  CGAL::Mesh_3::Polyline_visitor<Point_3, Graph> visitor(polylines, graph);
+  CGAL::split_graph_into_polylines(graph, visitor);
```

Comment:
Only the connectivity is copied and the vertices of the created graph have an internal property which is the original vertex. Thus properties are not copied.